r/R/dataset.R

```diff
@@ -15,6 +15,30 @@
 # specific language governing permissions and limitations
 # under the License.
 
+#' @usage NULL
+#' @format NULL
+#' @rdname Source
```

Comment:
This should be documented with Dataset, not Source, and it will need some prose added there